### DISCUSSION TITLE
kornia resize/rescale behavior different from cv2.resize

### DISCUSSION THREAD

**Discussion by busFred** [kornia/kornia#1839] [Questions]
As a background information, I have a trained and frozen neural net that is provided to me, and that model uses cv2 to preprocess input image. I am building an ad-hoc model that wraps around that frozen neural net. I need to feed the output of my model back to the given frozen model and then backward propagate with respect to the parameter of my model.

**With that said, I need to replicate the behavior of cv2 with kornia and I am having issue replicating `cv2.resize`**

Here are the three versions that I have tried, and I couldn't make the output of these three equal.

1. `cv2.resize(frame, newshape, interpolation=cv2.INTER_AREA)`
2. `kornia.geometry.rescale(new_obs,
                                          factor=(newshape[0] / obs.shape[2],
                                                  newshape[1] / obs.shape[3]),
                                          interpolation="area")`
3. `new_obs = kornia.geometry.resize(new_obs,
                                         newshape,
                                         interpolation="area")`

I am wondering what I am missing here.

**Comment by edgarriba** [comment]
hi! i've never compared this functionality against opencv. Internally the resize it uses pytorch `interpolate` which we cannot enforce to whatever opencv implements. In any case, could you share a quick colab to try reproduce?
